Some abi checker errors:

```
[INFO] --- abi-check-plugin:7-SNAPSHOT:abicheck (default) @ vespajlib ---
[ERROR] Class com.yahoo.tensor.Tensor$Cell: Extra method public java.lang.String toString(com.yahoo.tensor.TensorType)
[ERROR] Class com.yahoo.tensor.Tensor$Cell: Extra method public com.yahoo.tensor.Tensor$Cell detach()
[ERROR] Class com.yahoo.tensor.Tensor: Extra method public java.util.List largest()
[ERROR] Class com.yahoo.tensor.Tensor: Extra method public java.util.List smallest()
```

--- a/container-search/src/test/java/com/yahoo/search/rendering/JsonRendererTestCase.java
+++ b/container-search/src/test/java/com/yahoo/search/rendering/JsonRendererTestCase.java
@@ -150,7 +150,7 @@ public class JsonRendererTestCase {
                 + "                        \"tensor1\":{\"type\":\"tensor(x[3])\",\"cells\":[{\"address\":{\"x\":\"0\"},\"value\":1.5},{\"address\":{\"x\":\"1\"},\"value\":2.0},{\"address\":{\"x\":\"2\"},\"value\":2.5}]},"
                 + "                        \"tensor2\":{\"type\":\"tensor()\",\"cells\":[{\"address\":{},\"value\":0.5}]}"
                 + "                    },"
-                + "                    \"data\": \"...\""
+                + "                    \"data\": \"Data \\\\xc3\\\\xa6 \\\\xc3\\\\xa5\""
                 + "                },"
                 + "                \"id\": \"datatypestuff\","
                 + "                \"relevance\": 1.0"
@@ -178,7 +178,7 @@ public class JsonRendererTestCase {
         h.setField("tensor3", Tensor.from("{ {x:a, y:0}: 2.0, {x:a, y:1}: -1 }"));
         h.setField("object", new Thingie());
         h.setField("summaryfeatures", createSummaryFeatures());
-        h.setField("data", new RawData("Unknown artist from the moon".getBytes(StandardCharsets.US_ASCII)));
+        h.setField("data", new RawData("Data æ å".getBytes(StandardCharsets.UTF_8)));
         r.hits().add(h);
         r.setTotalHitCount(1L);
         String summary = render(r);

--- a/vespajlib/abi-spec.json
+++ b/vespajlib/abi-spec.json
@@ -1099,6 +1099,8 @@
       "public java.lang.Double setValue(java.lang.Double)",
       "public boolean equals(java.lang.Object)",
       "public int hashCode()",
+      "public java.lang.String toString(com.yahoo.tensor.TensorType)",
+      "public com.yahoo.tensor.Tensor$Cell detach()",
       "public bridge synthetic java.lang.Object setValue(java.lang.Object)",
       "public bridge synthetic java.lang.Object getValue()",
       "public bridge synthetic java.lang.Object getKey()"
@@ -1180,6 +1182,8 @@
       "public com.yahoo.tensor.Tensor sum()",
       "public com.yahoo.tensor.Tensor sum(java.lang.String)",
       "public com.yahoo.tensor.Tensor sum(java.util.List)",
+      "public java.util.List largest()",
+      "public java.util.List smallest()",
       "public abstract java.lang.String toString()",
       "public static java.lang.String toStandardString(com.yahoo.tensor.Tensor)",
       "public static java.lang.String contentToString(com.yahoo.tensor.Tensor)",

